DDSaaS: Metabase v1.1.0: Updated assets for self hosted instance support (#19984)
DataDog/integrations-core: metabase/README.md

8. Go to the **Instances** tab.
9. Click on **DNS alias** section to get the DNS alias value.

### Get self-hosted instance domain of Metabase (required for self-hosted instances only)

**Note**: Your self-hosted Metabase instance must be accessible from the internet via HTTPS only.

> 1. Log into your Metabase instance as an administrator.

Reviewer: ```suggestion
1. Log in to your Metabase instance as an administrator.
```

2. Go to **Settings > Admin Settings > General**.

> 3. Under **Site URL**, copy the domain from the URL. E.g. If the URL looks like `https://example.com`, then copy `example.com`.

Reviewer: ```suggestion
3. Under **Site URL**, copy the domain portion of the URL. For example, if the URL is `https://example.com`, copy `example.com`.
```

### Connect your Metabase account to Datadog

> 1. Add your Metabase instance type, Metabase DNS alias or Metabase self-hosted instance domain and Metabase API key    

Reviewer: ```suggestion
1. Add your Metabase instance type, DNS alias or self-hosted domain, and API key.
```

    |Parameters|Description|
    |--------------------|--------------------|

>     |Metabase instance type|The hosting type of your Metabase instance, i.e. `cloud` or `self-hosted`. Default is `cloud`.|

Reviewer: ```suggestion
    |Metabase instance type|The hosting type of your Metabase instance. Valid values are `cloud` or `self-hosted`. Default is `cloud`.|
```

>     |Metabase DNS alias|The DNS alias of your Metabase cloud instance (required for cloud instances only). Should be minimum 3 characters long (small case letters, dashes and numbers only).|

Reviewer: ```suggestion
    |Metabase DNS alias|The DNS alias of your Metabase cloud instance (required for cloud instances only). Must be at least three characters long and contain only lowercase letters, dashes, and numbers.|
```
